fix: LEAP-433: Fix cursor disappearence issue in labeling interface

juliosgarbi opened the pull request:
### PR fulfills these requirements
- [x] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [ ] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [x] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [x] Self-reviewed and ran all changes on a local instance (for bug fixes/features)



#### Change has impacts in these area(s)
- [ ] Product design
- [ ] Backend (Database)
- [ ] Backend (API)
- [x] Frontend



### Describe the reason for change
This PR addresses the cursor disappearance issue observed in the labeling interface, because canvas always has 100% width despite the image size and brushing cursor over the canvas is a grey-ish circle, practically invisible over lightgrey background.

![5a6a0f4a-10f3-43e0-8ab6-75bbf112e2fa](https://github.com/HumanSignal/label-studio/assets/4741623/079da451-86eb-478f-9d18-625cc895830c)


#### What does this fix?
The proposed solution is to introduce a thin black border around the cursor, enhancing its visibility against varying background contrasts.

![Screen-Recording-2024-04-15-at-13 04 58](https://github.com/HumanSignal/label-studio/assets/4741623/bd65ca6f-ef42-426e-91ac-b1ea9895f846)

juliosgarbi committed:
fix: LEAP-433: Fix cursor disappearence issue in labeling interface

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8692390293

yyassi-heartex commented on web/libs/editor/src/utils/canvas.js:
```diff
@@ -352,12 +352,22 @@ function Region2RLE(region) {
 function brushSizeCircle(size) {
   const canvas = document.createElement('canvas');
   const ctx = canvas.getContext('2d');
+  const canvasSize = size * 4 + 8;
+  const circlePos = size / 2 + 4;
```
I see this is copy pasted from before, but what is 4? what is 8? can we get those defined as constants?

juliosgarbi committed:
move magic numbers to const

juliosgarbi commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/8756196850)
> Successfully merged:  360 files changed, 907 insertions(+), 822 deletions(-)

juliosgarbi committed:
Merge branch 'develop' into 'fb-leap-433'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8756196850

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8756218599
